Modules/Microsoft365DSC/DSCResources/MSFT_AADRoleSetting/MSFT_AADRoleSetting.psm1

```diff
@@ -0,0 +1,1232 @@
+
+        [Parameter()]
+        [System.String[]]
+        $EligibleAlertNotificationAdditionalRecipient,
+
+        [Parameter()]
+        [System.Boolean]
+        $EligibleAlertNotificationOnlyCritical,
+
+        [Parameter()]
+        [System.Boolean]
+        $EligibleAssigneeNotificationDefaultRecipient,
+
+        [Parameter()]
+        [System.String[]]
+        $EligibleAssigneeNotificationAdditionalRecipient,
+
+        [Parameter()]
+        [System.Boolean]
+        $EligibleAssigneeNotificationOnlyCritical,
+
+        [Parameter()]
+        [System.Boolean]
+        $EligibleApproveNotificationDefaultRecipient,
+
+        [Parameter()]
+        [System.String[]]
+        $EligibleApproveNotificationAdditionalRecipient,
+
+        [Parameter()]
+        [System.Boolean]
+        $EligibleApproveNotificationOnlyCritical,
+
+        [Parameter()]
+        [System.Boolean]
+        $ActiveAlertNotificationDefaultRecipient,
+
+        [Parameter()]
+        [System.String[]]
+        $ActiveAlertNotificationAdditionalRecipient,
+
+        [Parameter()]
+        [System.Boolean]
+        $ActiveAlertNotificationOnlyCritical,
+
+        [Parameter()]
+        [System.Boolean]
+        $ActiveAssigneeNotificationDefaultRecipient,
+
+        [Parameter()]
+        [System.String[]]
+        $ActiveAssigneeNotificationAdditionalRecipient,
+
+        [Parameter()]
+        [System.Boolean]
+        $ActiveAssigneeNotificationOnlyCritical,
+
+        [Parameter()]
+        [System.Boolean]
+        $ActiveApproveNotificationDefaultRecipient,
+
+        [Parameter()]
+        [System.String[]]
+        $ActiveApproveNotificationAdditionalRecipient,
+
+        [Parameter()]
+        [System.Boolean]
+        $ActiveApproveNotificationOnlyCritical,
+
+        [Parameter()]
+        [System.Boolean]
+        $EligibleAssignmentAlertNotificationDefaultRecipient,
+
+        [Parameter()]
+        [System.String[]]
+        $EligibleAssignmentAlertNotificationAdditionalRecipient,
+
+        [Parameter()]
+        [System.Boolean]
+        $EligibleAssignmentAlertNotificationOnlyCritical,
+
+        [Parameter()]
+        [System.Boolean]
+        $EligibleAssignmentAssigneeNotificationDefaultRecipient,
+
+        [Parameter()]
+        [System.String[]]
+        $EligibleAssignmentAssigneeNotificationAdditionalRecipient,
+
+        [Parameter()]
+        [System.Boolean]
+        $EligibleAssignmentAssigneeNotificationOnlyCritical,
+
+        [Parameter()]
+        [System.Management.Automation.PSCredential]
+        $Credential,
+
+        [Parameter()]
+        [System.String]
+        $ApplicationId,
+
+        [Parameter()]
+        [System.String]
+        $TenantId,
+
+        [Parameter()]
+        [System.String]
+        $ApplicationSecret,
+
+        [Parameter()]
+        [System.String]
+        $CertificateThumbprint
+    )
+
+    Write-Verbose -Message "Setting configuration of Role settings"
```

Comment:
Please add the $DisplayName variable, to the verbose logging shows which Role is being processed.